plugins/woocommerce/includes/admin/settings/class-wc-settings-advanced.php

```diff
@@ -508,6 +508,8 @@ public function output() {
 	public function save() {
 		// phpcs:disable WordPress.Security.NonceVerification.Missing
 		global $current_section;
+		$prev_value = get_option( 'woocommerce_allow_tracking', 'no' );
```

Comment:
I have no problem updating it. However, at least in this part of the logic it seems unnecessary as we check after to see if `$prev_value` is 'yes' or 'no' explicitly.

Reply:
Yup, the suggestion about making it easier for comparisons was related to a suggestion I didn't end up posting because I was confused about checking tracking twice, once before saving and once after.

Either way though it helps to check the value in a consistent way, or use a helper function to keep it consistent.